bootstrap_cfn/fab_tasks.py

```diff
@@ -738,7 +748,14 @@ def cfn_create(test=False):
         # So delete the SSL cert that we uploaded
         if 'ssl' in cfn_config.data:
             iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
-        abort('Failed to create stack: {0}'.format(stack))
+        # delete the DNS tag
+        logger.info("Deleting DNS tag for '{stack} from '{zone}'...".format(stack=stack_id, zone=zone_name))
+        try:
+            print green("Removing tag stack from DNS...")
+            r53_conn.delete_txt_record(zone_name, zone_id, txt_tag_record)
```

Comment:
`r53_conn.delete_txt_record` only removes txt record. we probably need to delete/check Alias record too?